content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md

```diff
@@ -343,3 +343,11 @@ credentials you should have created when setting up your InfluxDB Cloud
 Dedicated cluster.
 Auth0 issues a short-lived (1 hour) token that authenticates access to your
 InfluxDB Cloud Dedicated cluster.
+
+
+When authenticating on Linux, if you see a warning on `stderr` like
+`Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load it.`,
+this can be prevented by either:
+- Unsetting the `GTK_MODULES` environment variable
+  (or at least removing `gail:atk-bridge` from it's value)
+- Removing the `libatk-adaptor` package and rebooting
```

Comment:
```suggestion
## Troubleshoot

- **Not loading module "atk-bridge"**: When authenticating, some Linux systems might report the following warning in the terminal (stderr):

  ```sh
  Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load it.
  ```

  To silence the warning, make one of the following changes to your system:
  
  - Unset the `GTK_MODULES` environment variable or remove `gail:atk-bridge` from it's value.
  - Remove the `libatk-adaptor` package and reboot.
```